templates/base.html

```diff
@@ -52,9 +55,15 @@
         data-readonly="{{ settings.READ_ONLY|json }}"
         data-media-url="{{ MEDIA_URL }}"
         data-static-url="{{ STATIC_URL }}"
+        {% if waffle.flag('restyle') %}
+          id="restyle"
```

Comment:
I'd make this a class, not an id. I like the idea that ids are for humans and classes are for computers.